CI-7271 documenting our upcoming mTLS support (#178)
AdobeDocs/adobe-io-events: src/pages/support/tracing.md

<InlineAlert variant="info" slots="text"/>
Note that the maximum size of 'request-body' and 'response-body' in a trace is 64 KB. If any of these fields are larger than 64 KB, then they are trimmed at 64 KB boundary. The delivered event payload is not trimmed.

## Access Debug Tracing

* Log into [Adobe Developer Console](https://developer.adobe.com/console/) and select the project containing the event registration.

> * From the *Project Overview*, select the event under *Events* in the left navigation.

Reviewer: ```suggestion
* From the *Project Overview* page, select the event registration under *Events* in the left navigation.
```

> * From the *Registration Details*, select the *Debug Tracing* tab. You will see a list of events, beginning with the most recent.

Reviewer: ```suggestion
* On the *Registration Details* page, click the *Debug Tracing* tab to see all event delivery traces, with the most recent entries shown first.
```

## Status Codes

Selecting a specific trace, you can also display the HTTP response (`body` and `codes`) received by Adobe IO Events HTTP Client from your webhook.

![](./img/events-debug-tracing-expanded_response.png)

These [HTTP response codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status) are three-digit numbers sent by **your** server 
to indicate the result of Adobe I/O Events HTTP client request. 
They are categorized into five classes: 
* `1xx` (Informational), 
* `2xx` (Successful), 
* `3xx` (Redirection), 
* `4xx` (Client Error), 
* `5xx` (Server Error)
*  On top of these, Adobe I/O Events uses a set of custom error codes in the `6xx` range (see below).

If the code is not `Successful` Adobe I/O Events HTTP client will retry see our [Retry Delivery](../guides/index.md#troubleshooting-unstabledisabled-registration-status) documentation for more details.

### `6XX` Custom Status Codes
To ease even further debugging, this Debug Tracing UI uses a set of custom status `6xx` codes to indicate specific issues encountered during event delivery, 
when no HTTP response was received from your webhook server.

These codes are

> * `601` Somewhat similar to `Not Implemented` HTTP `501` status code.

Reviewer: ```suggestion
* `601`: Similar to `Not Implemented` HTTP `501` status code.
```

  This error code indicates that Adobe I/O Events HTTP client
  failed with an unexpected error while trying to `POST` the event to the target webhook endpoint.

> * `602` Somewhat similar to `Bad Gateway` HTTP `502` status code.

Reviewer: ```suggestion
* `602`: Similar to `Bad Gateway` HTTP `502` status code.
```

  This status code indicates that Adobe I/O Events HTTP client 
  faced network issues or failures to connect to the target webhook endpoint

>   (errors such, `No Route To Host` or `Socket` exception).

Reviewer: ```suggestion
  (errors such as `No Route To Host` or `Socket` exception).
```

> * `604` Somewhat similar to `Gateway Timeout` HTTP `504` status code. 

Reviewer: ```suggestion
* `604`: Similar to `Gateway Timeout` HTTP `504` status code. 
```

  This status code indicates that your webhook server did not respond within the
  expected time frame. And that Adobe I/O Events HTTP client failed, timing out.

> * `611` Somewhat similar to `Network Authentication Required` HTTP `511` status code.

Reviewer: ```suggestion
* `611`: Similar to `Network Authentication Required` HTTP `511` status code.
```
